### ISSUE TITLE
Viewers: Line chat Y axis label is broken

### ISSUE THREAD

**Issue by dskatov** [datagrok-ai/public#657]
In a programmatically created, Line chart Y axis label is not showing, selecting different Y axis data is broken as well.

This needs a standalone reproduction code in JS.

**Comment by AufarZakiev** [comment]
Sample to reproduce:
```
const data = grok.data.demo.randomWalk(20, 2);
data.col('#0').name = 'Test column \n Units';
data.col('#1').name = 'Test column 2 \n Units';

let sp = DG.Viewer.lineChart(data, {
    xColumnName: 'Test column \n Units',
    yColumnNames: ['Test column 2 \n Units']
});
grok.shell.newView('View', [sp]);
```

Removing any all of `\n` symbols return to the normal behavior. The `\n` symbol support is required

**Comment by dionissqq** [comment]
Fixed in v1.6.10
